docs/metrics.md

```diff
@@ -0,0 +1,10 @@
+# Metrics
+
+## Prometheus Metrics
+
+Available Prometheus metrics:
+
+| Metric name    | Description                           |
+| -------------- | ------------------------------------- |
+| `actions_node` | Number of actions per node            |
+| `events_error` | Number of errors in events processing |
```

Comment:
If I understand correctly, we also emit two other metrics, defined in the [observability package](https://github.com/aws/aws-node-termination-handler/blob/main/pkg/observability/opentelemetry.go#L29-L35). Can you include those as well?

Reply:
I think those are the metric labels. The [registration function](https://github.com/aws/aws-node-termination-handler/blob/ccaae0544f0971ca80f501734fdb5b70952a3235/pkg/observability/opentelemetry.go#L106-L125) only registers the 2.